viYarn/Episode/view/EpisodeViewController.swift

```diff
@@ -8,14 +8,24 @@
 
 import UIKit
 
-class EpisodeTableViewController: UITableViewController {
+class EpisodeViewController: UIViewController, EpisodeLoaderDelagate {
+    
+    lazy var tableView: UITableView = {
+        let table:UITableView = UITableView()
```

Comment:
p0: type inference: we don't have to specify the type of the instance if we initialize it after the equals sign. Swift can do it for us.